assessments/guide/security-reviewer.md

```diff
@@ -1,35 +1,41 @@
 # Security reviewer
 
+## Lead reviewer
+The lead reviewer serves as the primary security assessment facilitator, often coordinating responses between the project and reviewers, ensuring communications are performed in a timely fashion as described in the [guide](./).  The lead reviwer also responsible for coordinating and recruiting additional reviwers as appropriate, reviewing and deliberating conflict of interest statements, and managing the documentation updates (self-assessment and final assessment) to the project's assessment folder.
+
+The lead reviewer is further tasked with performing the clarifying questions phase of the security assessment.  The lead reviewer may identify another reviewer to perform this task on their behalf; serving as their delegate or designee for that task.  
+
+## Reviewers
 Reviewers seek to understand the system and probe its security.
 Design level issues should be considered as well as any high
-level problems with the project’s setup and operation. The [process](./)
+level problems with the project’s setup and operation. The [guide](./)
 is intended to provide an independent analysis based primarily on documents
 provided by the project itself.
 
-Reviewers may look at code, but aren't required to do so. Review of
-documents includes asking questions to understand if there are hidden
+Reviewers may look at code, but aren't required to do so. Reviewers are encouraged to familiarize themselves with the contents of the repo, such as any instructions, guides, docs, or process information to better assist them in their function as a reviewer. 
+
+Reviewers are expected to ask questions to understand if there are hidden
 assumptions, underestimated risk, or design issues that harm or risk
 the security of the project.
 
+A reviewer my be delegated to perform specific tasks by the lead security reviewer in order to ensure the appropriate experience is leverage in conducting the activity.  This may include but is not limited to the clarifying questions phase.
```

Comment:
```suggestion
A reviewer may be delegated to perform specific tasks by the lead security reviewer in order to ensure the appropriate experience is leverage in conducting the activity.  This may include but is not limited to the clarifying questions phase.
```